segment-integration/src/main/java/com/urbanairship/extension/segment/SegmentAutopilot.java

```diff
@@ -50,11 +49,6 @@ public AirshipConfigOptions createAirshipConfigOptions(@NonNull Context context)
                     .setProductionAppSecret(preferences.getString(APP_SECRET, null));
         }
 
-        // Apply GCM Sender from Segment
```

Comment:
You still want to respect this. `builder.setFcmSender`

Reply:
Done.